README.rst

```diff
@@ -1,12 +1,11 @@
-HTTPie: human-friendly CLI HTTP client for the API era
+HTTPie: Human-friendly CLI HTTP client for the API era
 ######################################################
 
 HTTPie (pronounced *aitch-tee-tee-pie*) is a command-line HTTP client.
-Its goal is to make CLI interaction with web services as human-friendly as possible.
-HTTPie is designed for testing, debugging, and generally interacting with APIs & HTTP servers.
-The ``http`` & ``https`` commands allow for creating and sending arbitrary HTTP requests.
-They use simple and natural syntax and provide formatted and colorized output.
-
+Its goal is to make command-line interactions with web services as
+human-friendly as possible. The ``http`` & ``https`` commands create and
+send HTTP/S requests using a simple and natural syntax. Response data is
+colorized in an easy-to-read format.
```

Comment:
I prefer the original version of the introductory paragraph.

Reply:
I edited the opening paragraph to include all the info from the original text.